pulpcore/openapi/__init__.py

```diff
@@ -258,6 +258,21 @@ def resolve_serializer(self, serializer, direction):
 
         return component
 
+    def _get_response_bodies(self):
+        """
+        Handle response status code.
+        """
+        response = super()._get_response_bodies()
+        if (
+            self.method == "POST"
+            and issubclass(self.view.__class__, mixins.CreateModelMixin)
+            and "201" not in response
+        ):
+            response["201"] = response["200"]
+            del response["200"]
```

Comment:
If you want to remove the 200 on the way it can be:
```suggestion
            response["201"] = response.pop("200")
```